cmd/kpod/ps.go

```diff
@@ -33,18 +32,36 @@ type psOptions struct {
 	label   string
 }
 
-type psOutputParams struct {
-	ID         string `json:"id"`
-	Image      string `json:"image"`
-	Command    string `json:"command"`
-	CreatedAt  string `json:"created"`
-	RunningFor string `json:"running"`
-	Status     string `json:"status"`
-	Ports      string `json:"ports"`
-	Size       string `json:"size"`
-	Names      string `json:"names"`
-	Labels     string `json:"labels"`
-	Mounts     string `json:"mounts"`
+type psTemplateParams struct {
+	ID         string
+	Image      string
+	Command    string
+	CreatedAt  string
+	RunningFor string
+	Status     string
+	Ports      string
+	Size       string
+	Names      string
+	Labels     string
+	Mounts     string
+}
+
+// psJSONParams is only used when the JSON format is specified and is better for data processing from JSON.
```

Comment:
micro nit of a micro nit.  Split this comment into two lines if you need to make other changes, otherwise ignore.